plugin/reload/metrics.go

```diff
@@ -0,0 +1,17 @@
+package reload
+
+import (
+	"github.com/coredns/coredns/plugin"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Metrics for the reload plugin
+var (
+	FailedCount = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "reload",
+		Name:      "failed_count",
+		Help:      "A metric that counts the number of failed reload attempts.",
```

Comment:
Help: Counter of the number of failed reload attempts.

Reply:
Fixed